sandbox/public/perf/js/onBeforeEventSentHelper.js

```diff
@@ -0,0 +1,38 @@
+// code to collect tnta and trigger the Analytics Hit
+const trackingServer = "ujsl.sc.omtrdc.net";
+const reportSuite = "ujslecommerce";
+
+function triggerAnalyticsHit({ analyticsPayload, visitorID }) {
+  const url = `https://${trackingServer}/b/ss/${reportSuite}/0?g=${window.location}&r=${document.referrer}&vid=${visitorID}&tnta=${analyticsPayload}`;
```

Comment:
I think you need to apply @jonsnyder's comment here: vid -> mid. And do we need `pe:tnt`?